src/pages/workspace/GetAssistancePage.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {View, ScrollView} from 'react-native';
+import PropTypes from 'prop-types';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import KeyboardAvoidingView from '../../components/KeyboardAvoidingView';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import MenuItem from '../../components/MenuItem';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import RequestCallIcon from '../../../assets/images/request-call.svg';
+import * as Expensicons from '../../components/Icon/Expensicons';
+import * as Report from '../../libs/actions/Report';
+import ROUTES from '../../ROUTES';
+
+const propTypes = {
+    /** Route object from navigation */
+    route: PropTypes.shape({
+        params: PropTypes.shape({
+            /** The task ID to request the call for */
+            taskID: PropTypes.string,
+        }),
+    }).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const GetAssistancePage = props => (
+    <ScreenWrapper>
+        <KeyboardAvoidingView>
+            <HeaderWithCloseButton
+                title={props.translate('getAssistancePage.title')}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.goBack()}
+            />
+            <ScrollView style={styles.flex1} contentContainerStyle={[styles.pt0]}>
+                <View style={[styles.ph5]}>
+                    <View style={[styles.flex1, styles.flexRow, styles.alignItemsCenter]}>
+                        <Text style={[styles.h1, styles.flex1]}>
+                            {props.translate('getAssistancePage.subtitle')}
+                        </Text>
+                        <RequestCallIcon width={160} height={100} style={styles.flex1} />
```

Comment:
I'm not sure about the way this is being imported and used here. 
I think the preferred way to do this is to add that svg to [Illustrations.js](https://github.com/Expensify/App/blob/main/src/components/Icon/Illustrations.js).

Reply:
Actually, looping back to this, have you seen/considered using the [WorkspaceSection](https://github.com/Expensify/App/blob/main/src/pages/workspace/WorkspaceSection.js) component? 
Here's an [implementation example.](https://github.com/Expensify/App/blob/main/src/pages/workspace/bills/WorkspaceBillsFirstSection.js#L38-L52)

I think this is the preferred way to build these pages. It would end up encapsulating your MenuItem components below as well.

Reply:
Is this a "workspace" page + correct to place it in `/pages/workspace/` directory? Asking because it feels a bit more generic. If there are things we can reuse from `WorkspaceSection` that is cool, but we should maybe give it a more generic name unless we are considering this modal screen to be a "workspace" page.

Reply:
Yea, that's a good point. This isn't a workspace page per se. `WorkspaceSection` doesn't really seem to have workspace-specific funcitonality/styling. Maybe we should rename it to be more descriptive of what it really is (a title with an image on the side and some menu items). Do you think it's a good idea to rename it and use it here?

Reply:
>Do you think it's a good idea to rename it and use it here?

Depends on whether we are right about this:

>have you seen/considered using the WorkspaceSection component

Honestly, didn't look at that component to see if it is the same as is suggested - just trying to encourage consistent naming practices here. Feels like there's enough information here for @puneetlath to decide what to do.

Reply:
>I think the preferred way to do this is to add that svg to Illustrations.js.
Makes sense! I just followed the way it was on the `RequestCallPage` since this part of the page is the exact same. I'll update.

>Is this a "workspace" page + correct to place it in /pages/workspace/ directory? Asking because it feels a bit more generic.

Hm, yeah this is a good question. It's not really a page _of_ the workspace, but it can only be triggered from within a workspace today. I could imagine a future where it could be triggered from outside the workspace as well. I'm thinking that given that it's not specifically a page of the workspace that it makes sense to keep it in the main `/pages` folder instead of the `/pages/workspace` folder. 


>Actually, looping back to this, have you seen/considered using the WorkspaceSection component?
That's interesting. You're right that it would work well for both this page and the `RequestCall` page. Perhaps that's a good reason to make the `WorkspaceSection` component more generic. Perhaps something like `PageSectionWithIcon`

Reply:
I like the idea of generalizing the `WorkspaceSection` to be more generic. Could we call it `PageSectionWithIconAndMenu` or is that too verbose? That a menu is included in that component seems worth the mention. Unless there's a notion of a menu always being present in a "page section".

Reply:
Ok, I've gone ahead and moved this component out of the Workspace folder and made it a more generic `Section` component.